src/mscorlib/GenerateCompilerResponseFile.targets

```diff
@@ -16,7 +16,8 @@
       <Clean Include="$(IntermediateOutputPath)\moduleName.rsp" />
     </ItemGroup>
     <PropertyGroup>
-      <CompilerResponseFile>$(IntermediateOutputPath)\moduleName.rsp</CompilerResponseFile>
+      <CompilerResponseFile Condition="'$(CompilerResponseFile)'!=''">$(IntermediateOutputPath)\moduleName.rsp;$(CompilerResponseFile)</CompilerResponseFile>
```

Comment:
@joperezr can you change the name of the response file from `modulename.rsp` to `modulename.$(AssemblyName).rsp`? 

That way there won't be a race condition between the build of mscorlib and the build of System.Private.CoreLib.

The race condition problem is going to go away as soon as we merge the rename change but changing the name of the response file to be unique per assembly is still worth it.